src/java/org/apache/cassandra/config/CassandraDuration.java

```diff
@@ -147,7 +148,7 @@ private TimeUnit fromSymbol(String symbol)
         }
         throw new IllegalArgumentException(String.format("Unsupported time unit: %s. Supported units are: %s",
```

Comment:
Path not covered in `CassandraDurationTest`.

Reply:
Normally the invalid unit is caught early with the pattern, for now we don't hit this exception but it is good to have it in case of regressions later. For example If we add a unit to the pattern and we forget to acknowledge it in the `fromSymbol` method. Please check `testInvalidInputs`.

Reply:
I will add a few very simple test, to be on the safe side anyway